docs/StorIOSQLite.md

```diff
@@ -73,6 +73,25 @@ storIOSQLite
   });
 ```
 
+######Getting one result with Rx
+
+```java
+storIOSQLite
+  .get()
+  .listOfObjects(Tweet.class)
+  .withQuery(Query.builder()
+          .table("tweets")
+          .orderBy("date")
+          .build())
+  .prepare()
+  .createObservable() 
+  .take(1)  // To get the newest tweet
```

Comment:
I don't get this comment, why it'll get "newest tweet"? It'll get list of tweets once, not one tweet. Main idea — just get result once and don't receive updates of tweets table, you need to start using RxJava :)